packages/tracing/src/integrations/mongo.ts

```diff
@@ -47,7 +47,7 @@ const OPERATIONS = [
 const OPERATION_SIGNATURES: {
   [op in Operation]?: string[];
 } = {
-  aggregate: ['pipeline'],
+  // aggregate intentionally not included because `pipeline` arguments are too complex to serialize well
```

Comment:
```suggestion
  // aggregate intentionally not included because `pipeline` arguments are too complex to serialize well
  // see https://github.com/getsentry/sentry-javascript/pull/3102
```